fidelisnetwork.json

```diff
@@ -0,0 +1,1082 @@
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.total_alerts",
+                    "example_values": [
+                        512
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_ID",
+                    "example_values": [
+                        4046
+                    ],
+                    "column_name": "Alert ID",
+                    "column_order": 0,
+                    "data_type": "numeric",
+                    "contains": [
+                        "alert id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SEVERITY",
+                    "example_values": [
+                        "Medium"
+                    ],
+                    "column_name": "SEVERITY",
+                    "column_order": 1,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.HOST_IP",
+                    "example_values": [
+                        "10.10.10.10"
+                    ],
+                    "column_name": "HOST IP",
+                    "column_order": 2,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_TYPE",
+                    "example_values": [
+                        "Endpoint"
+                    ],
+                    "column_name": "ALERT TYPE",
+                    "column_order": 3,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.ALERT_TIME",
+                    "example_values": [
+                        "2022-04-27 06:12:45"
+                    ],
+                    "column_name": "ALERT TIME",
+                    "column_order": 4,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.aaData.*.SUMMARY",
+                    "example_values": [
+                        "Endpoint alert on root-win10: Artifact: GoogleUpdate.exe"
+                    ],
+                    "column_name": "SUMMARY",
+                    "column_order": 5,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.cancelled",
+                    "example_values": [
+                        false
+                    ],
+                    "data_type": "boolean"
+                },
+                {
+                    "data_path": "action_result.data.*.totalUnknown",
+                    "example_values": [
+                        false
```

Comment:
Should this value be a number?

Reply:
No, This is a boolean type output parameter.
<img width="394" alt="image" src="https://user-images.githubusercontent.com/85155176/165651097-3d5b3c60-dee8-4be6-a5fd-f0519291e768.png">